<AddItem /> - added possibility to show tooltip for all themes

egidijusja opened the pull request:
<!---
Thanks for submitting a pull request 😄 !
-->

<!---
- Be as descriptive as possible when explaining what was changed.
- Link to an issue if one exists
-->

### 🔦 Summary

...

<!--- Please mark all checkbox. If one is not relevant - delete it -->

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers <!--- Please tag the relevant team member -->
- [ ] 👨‍🎨 UX change is approved by the Design System UX <!--- Please tag the relevant team member -->
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

WixBuildServer commented:
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6666

EgoziE commented:
Can you please explain the scope of this PR?

egidijusja commented:
> Can you please explain the scope of this PR?
> AddItem has tooltip already.

Currently tooltip can be shown only when theme is `image`. I'm working on `showTitle` feature. We might need to show tooltip for other themes as well. Unfortunately there are some questions left, so we will have a meeting with UX guys. Made this PR as a draft.

egidijusja commented:
Hello @milkyfruit, we have added possibility to show tooltip for every theme on AddItem component as discussed earlier. Also added new example section `Title` with two examples:
1) Component with title.
2) Component with tooltip instead of title.

Right now there is possibility to have title and tooltip both at the same time on `AddItem` component. Should we add this example as well? Also can you please review this example in general (title, description, etc)

milkyfruit commented on packages/wix-style-react/src/AddItem/docs/index.story.js:
```diff
@@ -89,6 +89,12 @@ export default {
 
+          example({
+            title: 'Title',
+            text: 'AddItem title options',
```
@egidijusja please change text to: "AddItem title can be displayed as visible always or hidden in a tooltip. It's controlled with `children` and `tooltipContent` props."

milkyfruit reviewed (changes requested):
@egidijusja 

- [ ] please change the description text.
- [ ] When alignItems is set to "left" or "right" then tooltip's arrow appears in the opposite side that it should be:
![image](https://user-images.githubusercontent.com/1684587/106623071-e8bd4400-657c-11eb-8f32-5a791714cab3.png)

Unfortunately, I couldn't reproduce the issue when using tooltip in a playground with the same props: https://www.wix-style-react.com/?path=/story/introduction-playground--playground&snippet=7cecc445-faf8-40c8-8b5f-2cbcd2cd9d40

milkyfruit commented:
> Hello @milkyfruit, we have added possibility to show tooltip for every theme on AddItem component as discussed earlier. Also added new example section `Title` with two examples:
> 
> 1. Component with title.
> 2. Component with tooltip instead of title.
> 
> Right now there is possibility to have title and tooltip both at the same time on `AddItem` component. Should we add this example as well? Also can you please review this example in general (title, description, etc)

No need to show example with title and tooltip at once, it's an edge case.

mykas reviewed (approved):
Nice work!
